core/controllers/suggestion.py

```diff
@@ -481,6 +481,66 @@ class SuggestionListHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {}
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'suggestion_type': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': feconf.SUGGESTION_TYPE_CHOICES
+                },
+                'default_value': None
```

Comment:
Why is `None` default?

Reply:
ditto for all params below

Reply:
To my understanding, since there can be an arbitrary number of arguments for this handler, we must set a default value or else there will be an argument not found error. When I initially wrote the schema w/o a default value, it would raise that error.